symphony-sample/packer-templates/scripts/files/extras.repo

```diff
@@ -0,0 +1,4 @@
+[Extras]
+name=extras
+baseurl="http://mirrors.service.networklayer.com/centos/7/extras/x86_64/"
```

Comment:
This would only work if the build environment is an IBMCloud hosted machine

Reply:
changed to http://mirror.centos.org/centos/7/extras/x86_64/. Thanks, didn't think about this.